packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -59,48 +58,11 @@ const rainbowkitTheme = merge(darkTheme(), {
   }
 } as Theme)
 
-const AppContent = (): JSX.Element | null => {
+const ArbTokenBridgeStoreSyncWrapper = (): JSX.Element | null => {
+  const actions = useActions()
   const {
-    app: { connectionState }
+    app: { selectedToken }
   } = useAppState()
-  const [tosAccepted] = useLocalStorage<boolean>(TOS_LOCALSTORAGE_KEY, false)
-
-  useSyncQueryParamsToTestnetMode()
-
-  if (!tosAccepted) {
-    return null
-  }
-
-  if (connectionState === ConnectionState.NETWORK_ERROR) {
```

Comment:
we never set the `connectionState` to `ConnectionState.NETWORK_ERROR` in the code base so this case will never be hit